provision/acc_provision/acc_provision.py

```diff
@@ -2065,6 +2065,13 @@ def provision(args, apic_file, no_random):
     if flavor == "k8s-overlay":
         return True
 
+    if args.cko and prov_apic is not None:
+        apic = get_apic(config)
+        mcast_pool = config["aci_config"]["vmm_domain"]["mcast_range"]
+        configured_mcast_pool = apic.get_mcast_pool(config["aci_config"]["vmm_domain"]["mcast_pool"])
+        if mcast_pool != configured_mcast_pool:
```

Comment:
Can we info log here:

Requested mcast_pool <mcast_pool> does not match APIC configured <configured_mcast_pool>, using the latter